### PR TITLE
Adding Windows Operational Readiness container registry

### PR DESCRIPTION
**Pull request by knabben**
Adding a k8s official repository for the `kubernetes-sigs/windows-operational-readiness` project

### CODE REVIEW

File: infra/gcp/infra.yaml [hunk 1]
```diff
@@ -334,5 +334,6 @@ infra:
       k8s-staging-test-infra:
       k8s-staging-tg-exporter:
       k8s-staging-gmsa-webhook:
+      k8s-staging-op-readiness:
```
Comment (marosset) [inline infra/gcp/infra.yaml:337]: I know were pushing up against a character limit here but should we try and fit windows in the name?
maybe `k8s-staging-win-op-rdnss`?

File: k8s.gcr.io/manifests/k8s-staging-op-readiness/promoter-manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# google group for gcr.io/k8s-staging-op-readiness is kubernetes-sig-windows-leads@googlegroups.com
+registries:
+- name: gcr.io/k8s-staging-op-readiness
+  src: true
+- name: us.gcr.io/k8s-artifacts-prod/op-readiness
```
Comment (marosset) [inline k8s.gcr.io/manifests/k8s-staging-op-readiness/promoter-manifest.yaml:5]: windows-op-readiness?

**Comment by ameukam** [comment]
@knabben We should ensure a Google Group exists in https://github.com/kubernetes/k8s.io/blob/main/groups/sig-windows/groups.yaml and added to the restrictions file: https://github.com/kubernetes/k8s.io/blob/main/groups/restrictions.yaml#L191-L194.

**Comment by marosset** [comment]
LGMT from sig windows

**Comment by ameukam** [comment]
Project created:

```bash
...
      +- member: group:k8s-infra-staging-win-op-rdnss@kubernetes.io
      +  role: roles/storage.objectAdmin
       - member: projectEditor:k8s-staging-win-op-rdnss
         role: roles/storage.legacyBucketOwner
       - member: projectOwner:k8s-staging-win-op-rdnss
      @@ -4,6 +4,8 @@
         role: roles/storage.legacyBucketOwner
       - member: group:k8s-infra-artifact-admins@kubernetes.io
         role: roles/storage.objectAdmin
      +- member: group:k8s-infra-staging-win-op-rdnss@kubernetes.io
      +  role: roles/storage.legacyBucketWriter
       - member: group:k8s-infra-staging-win-op-rdnss@kubernetes.io
         role: roles/storage.objectAdmin
       - member: projectEditor:k8s-staging-win-op-rdnss
    Ensuring k8s-infra-staging-win-op-rdnss@kubernetes.io can use GCB in project: k8s-staging-win-op-rdnss 
    Updated IAM policy for project [k8s-staging-win-op-rdnss].
    @@ -1,5 +1,7 @@
     - member: group:k8s-infra-artifact-admins@kubernetes.io
       role: roles/viewer
    +- member: group:k8s-infra-staging-win-op-rdnss@kubernetes.io
    +  role: roles/cloudbuild.builds.editor
     - member: group:k8s-infra-staging-win-op-rdnss@kubernetes.io
       role: roles/viewer
     - member: serviceAccount:740513459330@cloudbuild.gserviceaccount.com
    Updated IAM policy for project [k8s-staging-win-op-rdnss].
    @@ -2,6 +2,8 @@
       role: roles/viewer
     - member: group:k8s-infra-staging-win-op-rdnss@kubernetes.io
       role: roles/cloudbuild.builds.editor
    +- member: group:k8s-infra-staging-win-op-rdnss@kubernetes.io
    +  role: roles/serviceusage.serviceUsageConsumer
     - member: group:k8s-infra-staging-win-op-rdnss@kubernetes.io
       role: roles/viewer
     - member: serviceAccount:740513459330@cloudbuild.gserviceaccount.com
    Ensuring gcb-builder@k8s-infra-prow-build-trusted.iam.gserviceaccount.com can use GCB in project: k8s-staging-win-op-rdnss 
    Updated IAM policy for project [k8s-staging-win-op-rdnss].
    @@ -8,6 +8,8 @@
       role: roles/viewer
     - member: serviceAccount:740513459330@cloudbuild.gserviceaccount.com
       role: roles/cloudbuild.builds.builder
    +- member: serviceAccount:gcb-builder@k8s-infra-prow-build-trusted.iam.gserviceaccount.com
    +  role: roles/cloudbuild.builds.builder
     - member: serviceAccount:k8s-infra-gcr-vuln-scanning@k8s-artifacts-prod.iam.gserviceaccount.com
       role: roles/containeranalysis.occurrences.viewer
     - member: serviceAccount:service-740513459330@containerregistry.iam.gserviceaccount.com
    @@ -14,3 +14,5 @@
       role: roles/storage.legacyBucketOwner
     - member: projectViewer:k8s-staging-win-op-rdnss
       role: roles/storage.legacyBucketReader
    +- member: serviceAccount:gcb-builder@k8s-infra-prow-build-trusted.iam.gserviceaccount.com
    +  role: roles/storage.objectCreator
    @@ -16,3 +16,5 @@
       role: roles/storage.legacyBucketReader
     - member: serviceAccount:gcb-builder@k8s-infra-prow-build-trusted.iam.gserviceaccount.com
       role: roles/storage.objectCreator
    +- member: serviceAccount:gcb-builder@k8s-infra-prow-build-trusted.iam.gserviceaccount.com
    +  role: roles/storage.objectViewer
Configuring special cases for Release Managers 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-artifact-promoter 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-build-image 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-ci-images 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-cip-test 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-experimental 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-kubernetes 
  Empowering kubernetes-release-test GCB service account to admin GCR 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-releng 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-releng-test 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-publishing-bot 
Done
```

**Pull request merged**